pom.xml

```diff
@@ -295,7 +296,7 @@
         <solr.httpclient.version>4.5.12</solr.httpclient.version>
         <solr.httpcore.version>4.4.13</solr.httpcore.version>
         <solr.httpmime.version>4.5.12</solr.httpmime.version>
-        <solr.log4j.version>2.13.2</solr.log4j.version>
```

Comment:
Unsure of the effect of this, I left it alone since there wasn't a solr update yet, but if it works then it should be fine...wonder why we ever kept them separate.

Reply:
It looks like this is only be used for test scope dependencies